sunpy/instr/goes.py

```diff
@@ -991,23 +992,14 @@ def _calc_rad_loss(temp: u.MK, em: u.cm**-3, obstime=None, force_download=False,
         if len(obstime) != n:
             raise IOError("obstime must have same number of elements as "
                           "temp and em.")
-        if type(obstime) == pandas.DatetimeIndex:
-            obstime = obstime.to_pydatetime
```

Comment:
:open_mouth:

Reply:
can `parse_time` not do this? #confused

Reply:
Oh. Yes. I was surprised that the old code assigned `obstime` to a function `to_pydatetime`. No one use `DatetimeIndex` with sunpy, I guess.